README.md

```diff
@@ -127,6 +173,43 @@ On the Camera Preview Page, the same second button will toggle between
 the full resolution camera view and a one-to-one pixel view intended
 to aid in focusing the camera (the camera is focused manually).
 
+# Dencam Pages
+
+The Dencam user interface has 5 pages, which are traversed using the 
+top button. Each page is displaying information designated to itself.
+(i.e. Solar Display Page displays information recieved from Sun Saver)
+
+## Networking Information Page
+
+This page displays Raspberry Pi username, wLan0 IP address, and the name of 
```

Comment:
"username" should be "hostname"
could make sense to say "hostname of the device" instead of "Raspberry Pi hostname" and abstract away the fact that the specific SBC that the controls the device is a Raspberry Pi.